src/SlateAsInputEditor/index.js

```diff
@@ -36,8 +36,8 @@ const EditorWrapper = styled.div`
   min-height: 750px;
   max-width: ${props => props.width || 'none'};
   min-width: ${props => props.width || 'none'};
-  border: 1px solid #979797;
-  border-radius: 10px;
+  border: ${props => props.EDITOR_BORDER || ' 1px solid #979797'};
+  box-shadow: ${props => props.EDITOR_SHADOW || ' 1px 2px 4px rgba(0, 0, 0, .5)'};
```

Comment:
I think we still want `border-radius`, to optionally make it `10px` for example. @michael-grover @DianaLease thoughts?

Reply:
yeah, different applications of this editor have different border radiuses

Reply:
Should I go ahead and add an optional border-radius for the editor?

Reply:
@pseudonerd yes please!

Reply:
I have updated the PR.